constrained-voucher.mkd

```diff
@@ -362,13 +362,15 @@ it provisionally accepts this CA certificate as if it were the result of "CA cer
 
 ### Registrar Extensions
 
-The Content-Format 50 MAY be supported and 60 MUST be supported by the Registrar for the /vs and /es resources.
+The Content-Format 50 (application/json) MAY be supported and 60 (application/cbor) MUST be supported by the Registrar for the /vs and /es resources.
```

Comment:
I had to look up CF 50 and 60.
/vs is Voucher-Status, and /es is Enrollment-Status.
RFC8995 defines it for JSON only.
We have examples for this in CBOR, but we don't have any clear text saying that the CDDL should be mapped directly to CBOR.  
We need to do that.